### PR TITLE
add device parameters with processing

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/constants/deviceParameters.ts [hunk 1]
```diff
@@ -2,6 +2,26 @@
  * Device parameters. Used in SetParameter/GetParameter.
  */
 
+/**
+ * Interval of data sending from device.
+ */
```
Comment (DarkPark) [inline src/constants/deviceParameters.ts:7]: as all these comments are rather simple what about one-liners?

```suggestion
/** interval of data sending from device */
```

File: src/CommandBinaryBuffer.ts [hunk 2]
```diff
@@ -143,24 +143,65 @@ export interface IEventMtxStatus {
 }
 
 
+/**
+ * Device send data periodically using this interval.
+ * TODO:
+ */
+// interface IParameterDataSendingInterval {
```
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply src/CommandBinaryBuffer.ts:150]: will be used in near future

File: src/CommandBinaryBuffer.ts [hunk 3]
```diff
@@ -143,24 +143,65 @@ export interface IEventMtxStatus {
...
+    type: number
+}
+
 /**
  * Initial values for pulse devices.
+ * deviceParameters.INITIAL_DATA = `23`.
  */
 interface IParameterInitialData {
     /** 4 byte int BE */
     value: number
```
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply src/CommandBinaryBuffer.ts:194]: fixed

File: src/constants/deviceParameters.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/constants/deviceParameters.ts:7]: don't like this, but ok, fixed

File: src/CommandBinaryBuffer.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply src/CommandBinaryBuffer.ts:150]: device parameter already added, encode/decode not ready due to problems with understanding that parameter values

**Review by DarkPark — APPROVED** [review]

**Pull request merged**